src/main/kotlin/com/thelastpickle/tlpstress/ProfileRunner.kt

```diff
@@ -105,7 +131,11 @@ class ProfileRunner(val context: StressContext,
             val op : Operation = if(readRate * 100 > ThreadLocalRandom.current().nextInt(0, 100)) {
                 runner.getNextSelect(key)
             } else {
-                runner.getNextMutation(key)
+                if(deleteRate * 100 > ThreadLocalRandom.current().nextInt(0, 100)) {
```

Comment:
I think if you pass `--delete .1` then you should get 10% of your ops as deletes, and making it a subset of mutations is hard to understand.  Let's extract `ThreadLocalRandom.current().nextInt(0, 100)` to a temporary variable and use something like the following:

```
val nextOp = ThreadLocalRandom.current().nextInt(0, 100)
if(readRate * 100 < nextOp) {
// reads
}
else if(readRate * 100 + deleteRate * 100 < nextOp) {
// deletes
}
else {
// other mutations
}

Reply:
I think i botched the < and > but I think you get the idea.

Reply:
yes, you nailed it. It's giving the right distribution this way.